### PR TITLE
Improve-home-page

### PR DESCRIPTION
**Pull request by sr-meh**
# ISSUE #58 
# Description

- Enhanced Product Table Display: The pull request includes enhancements to the display of product tables for both mobile and desktop views.

- Button Styles: Button styles have been added to improve the overall appearance of buttons within the tables. This can make the interface more visually appealing and user-friendly.

- Codebase Cleanup: Duplicate partials have been removed from the codebase. This cleanup helps maintain code consistency and efficiency, making it easier to maintain and understand the code.

- Footer Links Correction: The links in the footer have been corrected to direct users to the respective "About Us" and "Frequently Asked Questions" views. This ensures that users can easily access important information.

- Image Placeholder: The "no image" text has been replaced with the appropriate image placeholder. This change improves the visual representation of products, even when images are not available.

- Additionally,  modify the home page index to show a banner corresponding to each of the registered brands and display the associated products for each of them.

## Screenshots

image

### CODE REVIEW

File: app/views/caps/index.html.erb [hunk 1]
```diff
@@ -1,43 +1,4 @@
...
-  </div>
-  <% if current_user.usertype == 'admin'%>
-    <div class="text-center my-3">
-      <%= link_to "New product", new_product_path, class: "btn custom-button text-uppercase fw-bold" %>
-    </div>
-  <% end %>
+<div class="row">
+  <h1 class="my-3 login-heading fw-medium fs-1 fw-light text-center">CAPS</h1>
 </div>
+<%= render partial: "partials/Products/products" %>
```
Comment (JuanequeX) [inline app/views/caps/index.html.erb:4]: For partials, the folder should be named `shared`

File: app/views/partials/Products/_brands.html.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<div class="container">
+  <% @brands.each do |brand| %>
+    <% if brand.images.attached? %>
+      <% brand.images.each do |image| %>
+        <%= image_tag image, class: "img-fluid" %>
+      <% end %>
+    <% end %>
+
+    <div class="row mb-5">
+      <% caps_products_by_brand = @caps_products.select { |product| product.brand == brand } %>
```
Comment (JuanequeX) [inline app/views/partials/Products/_brands.html.erb:10]: This logic should separated in the controller.

**Review by claunicole — APPROVED** [review]

File: app/views/partials/Products/_brands.html.erb (hunk 2 above)
Comment (JuanequeX) [reply app/views/partials/Products/_brands.html.erb:10]:
```suggestion
     def products_by_brand
         brand = params[:brand] # You can get the brand from a parameter, for example, if is necessary!!
         @caps_products_by_brand = @caps_products.select { |product| product.brand == brand }
    end
```

File: app/views/caps/index.html.erb (hunk 1 above)
Comment (JuanequeX) [reply app/views/caps/index.html.erb:4]: and this same suggestion will be applied in each render partial!

**Review by JuanequeX — CHANGES_REQUESTED** [review]

**Review by JuanequeX — APPROVED** [review]

**Pull request merged**